source/user-manual/manager/configuring-cluster/advanced-settings.rst

```diff
@@ -1,27 +1,735 @@
+            [NOTICE]   (13231) : path to executable is /usr/sbin/haproxy
+            [ALERT]    (13231) : config : parsing [/etc/haproxy/haproxy.cfg:3] : 'pidfile' already specified. Continuing.
+
+.. _haproxy_helper_setup:
+
+HAProxy helper
+==============
+
+This is an optional tool to manage HAProxy configuration depending on the Wazuh cluster status in real-time.
+It provides the manager with the ability to automatically balance the agent TCP sessions.
+
+Some of its key features are:
+
+* Add/remove new servers to the Wazuh backend (1514/tcp) when detecting changes on the Wazuh cluster (e.g. new workers connected).
+* Balance excess agents per node when adding new servers to the Wazuh backend.
+* Balance agents when detecting an imbalance that exceeds the given tolerance.
+
+.. thumbnail:: /images/manual/cluster/haproxy-helper-architecture.png
+    :title: HAProxy helper architecture
+    :alt: HAProxy helper architecture
+    :align: center
+    :width: 80%
+
+
+The helper runs in an independent thread, that initiates with the ``wazuh-cluster`` daemon, and completes the next flow:
+
+.. thumbnail:: /images/manual/cluster/haproxy-helper-flow.png
+    :title: HAProxy helper flow
+    :alt: HAProxy helper flow
+    :align: center
+    :width: 80%
+
+
+How to enable it
+----------------
+
+.. note::
+    The recommended version of HAProxy is the 2.8 LTS.
+
+To use this feature is required to have a :ref:`HAProxy <haproxy_installation>` instance balancing the cluster using the **least connections** algorithm.
+
+Dataplane API configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    The Dataplane API is used by the helper to communicate with HAProxy and update the configuration according to the changes in the Wazuh cluster.
+
+    This is the basic configuration:
+
+    .. code-block:: yaml
+
+        dataplaneapi:
+            host: 0.0.0.0
+            port: 5555
+            transaction:
+                transaction_dir: /tmp/haproxy
+            user:
+            - insecure: true
+              password: <DATAPLANE_PASSWORD>
+              name: <DATAPLANE_USER>
+        haproxy:
+            config_file: /etc/haproxy/haproxy.cfg
+            haproxy_bin: /usr/sbin/haproxy
+            reload:
+                reload_delay: 5
+                reload_cmd: service haproxy reload
+                restart_cmd: service haproxy restart
+
+    To enable it will depend on the :ref:`installation method <haproxy_installation>`.
+
+    .. tabs::
+        .. group-tab:: Packages
+
+            1. Download the binary file for the installed HAProxy version
+
+            .. code-block:: console
+
+                # curl -sL https://github.com/haproxytech/dataplaneapi/releases/download/v2.8.X/dataplaneapi_2.8.X_linux_x86_64.tar.gz | tar xz | cp dataplaneapi /usr/local/bin/
```

Comment:
The url contains `.X`, I guess those should be replaced. Also, cp must be done in a different command, not using a pipe:
```suggestion
                # curl -sL https://github.com/haproxytech/dataplaneapi/releases/download/v2.8.7/dataplaneapi_2.8.7_linux_x86_64.tar.gz | tar xz && cp dataplaneapi /usr/local/bin/
```